src/components/Wishlist/Wishlist.jsx

```diff
@@ -10,7 +10,7 @@ const Wishlist = () => {
                         <div className="card-image-container">
                             <img
                                 className="responsive-img"
-                                src={require('../../assets/images/Fiddle-leaf.jpg')}
```

Comment:
instead of this put your assets folder in the public folder thereafter you would be to access the image  just by a slash for example here the image would be accessed with ('/assets/images/Fiddle-leaf.jpg') by any image tag(only) throughout the app

Reply:
It is generally believed that assets should not be added to the public folder. I might have to look more on this but why not `/src/assets/` ?

Reply:
Tanay also suggested to put the images in public folder in one of his reviews. Have to look for the reason though.